src/scripts/main.js

```diff
@@ -1 +1,71 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+const notification = (message, type) => {
+  const div = document.createElement('div');
+
+  div.classList.add('notification', type);
+  div.innerText = message;
+  div.dataset.qa = 'notification';
+
+  body.append(div);
+};
+
+const messageS1 = () => {
+  notification('First promise was resolved', 'success');
+};
+
+const messageS2 = () => {
+  notification('Second promise was resolved', 'success');
+};
+
+const messageS3 = () => {
+  notification('Third promise was resolved', 'success');
+};
+
+const messageE1 = () => {
+  notification('First promise was rejected', 'warning');
+};
```

Comment:
I figure out that "S" means "success" and "E" means "error", but you should not name functions in that way. There's no need to create these functions as you don't re-use them, you can call "notification" where you need to.